relay-base-schema/src/data_category.rs

```diff
@@ -57,6 +57,12 @@ pub enum DataCategory {
     /// but we define it here to prevent clashing values since this data category enumeration
     /// is also used outside of Relay via the Python package.
     MonitorSeat = 13,
+    /// User Feedback
+    ///
+    /// Represents a User Feedback processed.
+    /// Currently standardized on name UserReportV2 to avoid clashing with the old UserReport.
+    /// TODO(jferg): Rename this to UserFeedback once old UserReport is deprecated.
+    UserReportV2 = 14,
```

Comment:
Separate question: Why do we need a new data category, if user feedback items are converted to events (i.e. `Error` category)?

Reply:
they are not going to be Error category events. They will be billed separately.